### PR TITLE
STAR-492 side menu

### PR DESCRIPTION
**Pull request by oleynichenko**
st1
st2
st3

### CODE REVIEW

File: src/components/page/links.js [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+  },
+  {
+    title: 'menu:statistics',
+    to: `/event/${id}/statistics`,
+    icon: faChartBar,
+  },
+];
+
+const Links = ({ selectedEvent = {} }) => {
+  const links = useMemo(() => getLinks(selectedEvent.id), [selectedEvent]);
```
Comment (vitalii-retel) [inline src/components/page/links.js:78]: Dependency should by on `id`, I don't think useMemo checks the whole object, guess it checks only reference. But here we need `id` only.

**Review by vitalii-retel — COMMENTED** [review]
- can we use em instead of px everywhere?
- please take a look at how we can test it already

File: src/components/page/links.js (hunk 1 above)
Comment (oleynichenko) [reply src/components/page/links.js]: fixed

**Comment by oleynichenko** [comment]
> * can we use em instead of px everywhere?
fixed but not everywhere. For elements that do not depend on text I propose to keep px.

> * please take a look at how we can test it already
added for menu

**Review by vitalii-retel — APPROVED** [review]

**Pull request merged**